_redirects

```diff
@@ -38,7 +38,21 @@
 /getting-started/guides/*    /docs/getting-started/livechat-apps/:splat  301 
 /authorization/livechat-accounts/*    /docs/authorization/global-accounts/:splat  301 
 /authorization/livechat-accounts-api/*    /docs/authorization/global-accounts-api/:splat  301 
-/messaging/apple-business-chat/*    /docs/messaging/apple-messages-for-business/:splat  301 
+/messaging/apple-business-chat/*    /docs/messaging/apple-messages-for-business/:splat  301
+/changelog/  /docs/messaging/ 301
+/chat-widget-moments/   /docs/extending-chat-widget/chat-widget-moments/ 301
+/chat-widget-themes/   /docs/extending-chat-widget-customer-sdk 301
+/management/getting-started/guides/webhook-apps/*    /docs/getting-started/livechat-apps/webhook-apps/:splat 301
+/management/webhooks/changelog/*    /docs/management/changelog/:splat 301
+/management/webhooks/v3.3/changelog/*    /docs/management/changelog/:splat 301
+/messaging/rtm-reference/   /docs/messaging/ 301
+/messaging/agent-chat-api/v3.3/rtm-reference/messaging/agent-chat-api/rtm-pushes/    /docs/messaging/agent-chat-api/rtm-pushes 301 
```

Comment:
@jakubsikora, is it OK to have this many "edge case" redirects?

Reply:
Yep no problem